integration-tests/src/test/resources/sitconfig/configoverrides/jdbc-JdbcTestDataSource-0.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
missing copyright notice

Reply:
Added.

Reply:
Because of the errors I had to remove the copyright notice in config override files.